subjects/DPxAI/first-hello/README.md

```diff
@@ -0,0 +1,85 @@
+## First hello
+
+> JSPowered Mode
+
+### Context
+
+Your robot winked, but you can make it talk too! or at least, make it say his first hello to the world!
+
+> Follow the instructions, ask your peers if you are stuck and stay motivated because you are close to the Shape Crafting goal!
```

Comment:
Nothing major, but `Shape Crafting` is the name of the quest 00 and this exercise is linked to the quest 02

Reply:
Fixed.